examples/plot_missing_values.py

```diff
@@ -4,11 +4,11 @@
 ====================================================
 
 Missing values can be replaced by the mean, the median or the most frequent
-value using the basic :func:`sklearn.impute.SimpleImputer`.
+value using the basic :class:`sklearn.impute.SimpleImputer`.
 The median is a more robust estimator for data with high magnitude variables
 which could dominate results (otherwise known as a 'long tail').
 
-Another option is the :func:`sklearn.impute.ChainedImputer`. This uses
+Another option is the :class:`sklearn.impute.ChainedImputer`. This uses
```

Comment:
Looks like you missed renaming `ChainedImputer` to `IterativeImputer` here.

Reply:
Ah, thanks.